nimsuggest/tests/taccent_highlight.nim

```diff
@@ -0,0 +1,8 @@
+proc `$$$`#[!]#
+
+discard """
+disabled:true
```

Comment:
Disabled true?

Reply:
It's because `highlight` output differs between EPC and "normal" mode. I can't make the test work for both.

Reply:
Meh alright, but we need to change this, eventually.